tests/core/test_core.py

```diff
@@ -551,22 +551,28 @@ def test_maximum_popup_dimensions(
     @pytest.mark.parametrize(
         "active_conversation_info",
         [
-            case({"sender_name": "hamlet"}, id="in_pm_narrow_with_sender_typing:start"),
-            case({}, id="in_pm_narrow_with_sender_typing:stop"),
+            case({"hamlet@zulip.com"}, id="in_pm_narrow_with_sender_typing:start"),
+            case(set(), id="in_pm_narrow_with_sender_typing:stop"),
         ],
     )
     def test_show_typing_notification(
         self,
         mocker: MockerFixture,
         controller: Controller,
-        active_conversation_info: Dict[str, str],
+        active_conversation_info: Set[Any],
```

Comment:
Check the types across commits - whether we settle on Set[str] as now or Set[int] for user ids, we shouldn't need to use Any now.